studio/completion_service/completion_service.py

```diff
@@ -154,9 +152,8 @@ def __exit__(self, *args):
 
     def close(self, delete_queue=True):
         self.logger.info("Studioml completion service shutting down")
-        # if self.queue_name != 'local' and delete_queue:
         request_delete_queue = self.shutdown_del_queue or delete_queue
-        self.queue.shutdown(request_delete_queue)
```

Comment:
Route RMQ operations through runner module, which further sends them to RMQ queues cache.